src/main.rs

```diff
@@ -47,30 +48,40 @@ enum Commands {
     },
     /// Deploy a new testnet environment using the latest version of the safenode binary.
     Deploy {
-        /// Optionally supply the name of a branch on the Github repository to be used for the
```

Comment:
Minor suggestion: I think it's useful to have the arguments sorted in alphabetical order rather than arbitrarily.

Reply:
I organized it based on the `required, optional args` + `bundled together related args`. It is a bit arbitrary, but I was able to follow it better as the related ones were close by. 

I can change it, if it's hard to follow.

Reply:
Nah it's ok, not a problem.